editors/code/src/client.ts

```diff
@@ -30,15 +30,14 @@ export function configToServerOptions(config: Config) {
     };
 }
 
-export async function createClient(config: Config, serverPath: string): Promise<lc.LanguageClient> {
+export async function createClient(config: Config, serverPath: string, workspaceFolder: vscode.WorkspaceFolder): Promise<lc.LanguageClient> {
```

Comment:
```suggestion
export async function createClient(config: Config, serverPath: string, cwd: string): Promise<lc.LanguageClient> {
```

All we do with ws folder is extracting the path, so we as well might just accept the path in the first place.